tools/dotcms-cli/cli/src/main/java/com/dotcms/cli/command/files/FilesPull.java

```diff
@@ -155,4 +155,9 @@ public Optional<String> getCustomMixinName() {
         return Optional.empty();
     }
 
+    @Override
+    public int getOrder() {
+        return 3;
```

Comment:
Wouldn't it be better if we make this into an enum
public enum ExecOrder {
    SITE(1), CONTENT_TYPE(2).. 
}
and then simply return order?